xml/art_suseconnect_tracking.xml

```diff
@@ -0,0 +1,405 @@
+   </para>
+  </note>
+
+  <sect2 xml:id="disabling-suseconnect-timer">
+   <title>Disabling the keep-alive timer</title>
+   <para>
+    If you prefer to not have the &suseconnect; keep-alive timer running on your system, you can
+    disable it with <command>systemctl</command>:
+   </para>
+<screen>&prompt.sudo;systemctl disable --now suse-connect-keepalive.timer</screen>
+   <para>
+    Once the timer is disabled, subsequent updates to the &suseconnect; package will not re-enable
+    it.
+   </para>
+   <tip>
+    <title>Re-enabling the keep-alive timer</title>
+    <para>
+     If you decide to re-enable the disabled keep-alive timer, run the following command:
+    </para>
+<screen>&prompt.sudo;systemctl enable --now suse-connect-keepalive.timer</screen>
+   </tip>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="identifying-inactive-systems-with-registration-servers">
+  <title>Identifying inactive systems</title>
+
+  <para>
+   When &suseconnect; reports details about active hosts daily, SCC or RMT collect this information
+   and let you filter out registered inactive hosts. If your hosts are registered against RMT or
+   &suma;, these registration servers forward the received information to SCC.
+  </para>
+
+  <sect2 xml:id="identifying-inactive-systems-with-scc">
+   <title>Identifying inactive systems with SCC</title>
+   <para>
+    In the SCC Web user interface, you can limit the list of systems to see only inactive
+    hosts.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Log in to SCC at <link xlink:href="https://scc.suse.com"/> and select your organization in the
+      left pane.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select <guimenu>Systems</guimenu> from the top menu to list all registered hosts by default.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select <guimenu>Inactive</guimenu> from the top left drop-down list.
+     </para>
+     <figure>
+      <title>Drop-down list with a filter for inactive hosts</title>
+      <mediaobject>
+       <imageobject role="fo">
+        <imagedata fileref="scc-inactive-filter.png" width="45%"/>
+       </imageobject>
+       <imageobject role="html">
+        <imagedata fileref="scc-inactive-filter.png" width="45%"/>
+       </imageobject>
+      </mediaobject>
+     </figure>
+    </step>
+    <step>
+     <para>
+      If you have sufficient privileges and your inactive host is registered directly with SCC, you
+      can de-register it by clicking the three dots on its right and selecting
+      <guimenu>De-register</guimenu>.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="identifying-inactive-systems-with-rmt">
+   <title>Identifying inactive systems with RMT</title>
+   <para>
+    By using the <command>rmt-cli purge</command> command, you can view and remove hosts that have not
+    contacted the RMT server since a given date in the past:
```

Comment:
I'd also write a small note detailing that if the `--before` option is not passed, then we assume 3 months from now. I'd also write about the `--no-confirmation` option, which allows user to simply purge everything without needing to confirm afterwards.

Reply:
Thank you for reviewing the doc, Miquel!

Regarding the additional options, since this doc is not an "RMT" doc, my suggestion I would document them in RMT's documentation (if not done already) and link it here, rather than duplicate the info. Thoughts?

Reply:
@lagartoflojo i agree. Can you create a bug/JIRA issue to document these new options in RMT please?

Reply:
Done https://bugzilla.suse.com/show_bug.cgi?id=1202752

Reply:
Agreed 👍🏻